src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -1073,6 +1088,11 @@ def create(  # pylint: disable=too-many-locals,too-many-arguments,too-many-state
             storage_location = storage_override
 
         sql_backend.execute(ddl)
+
+        # CTAS AND NON_DELTA does not support TBLPROPERTIES
```

Comment:
Really? Why?

Reply:
<img width="993" alt="image" src="https://github.com/databrickslabs/ucx/assets/136742693/a76b35d6-1bcb-4058-88e7-da5079dbbd87">

Reply:
It is failing with SYNTAX error.
<img width="993" alt="image" src="https://github.com/databrickslabs/ucx/assets/136742693/f9ab8960-77c3-4cca-bc29-6ef390e50034">

Reply:
Can it be done in a second SQL query?